src/components/animateNumber.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+<h3>{{displaynumber(displayNumber).first}}<span class="highlighted">{{displaynumber(displayNumber).second}}</span> </h3>
+</template>
+
+<script>
+export default {
+  props: {'number': {default: 0}},
+  data () {
+    return {
+      displayNumber: 0,
+      interval: false
+    }
+  },
+
+  ready: function () {
+    this.displayNumber = this.number ? this.number : 0
+  },
+
+  watch: {
+    number: function () {
+      clearInterval(this.interval)
+      if (this.number === this.displayNumber) {
+        return
+      }
+
+      this.interval = window.setInterval(function () {
+        if (this.displayNumber !== this.number) {
+          var change = (this.number - this.displayNumber) / 30
+          change = change >= 0 ? Math.ceil(change) : Math.floor(change)
+          this.displayNumber = this.displayNumber + change
+        }
+      }.bind(this), 80)
+    }
+  },
+  methods: {
+    displaynumber (num) {
+      num = num.toString()
+      return {
+        first: num.substr(0, 1),
+        second: num.substr(1, num.length)
+      }
+    }
+  }
+}
+</script>
+
+<style scoped>
+.highlighted{
+  color:#fa631c !important;
+}
+h3{
+  color:white;
+  margin: 0 !important;
```

Comment:
Do not use !important everywhere. Only use it if it is required. And use consistent spacing. e.g.,
h3 {
    color: white;
}